### PR TITLE
Backport master into release/13

### PR DESCRIPTION
**Pull request by TrueBrain**
## Description
Backport of all closed Pull Requests labeled as 'backport requested' into `release/13`.
- https://github.com/OpenTTD/OpenTTD/pull/10879
- https://github.com/OpenTTD/OpenTTD/pull/10919
- https://github.com/OpenTTD/OpenTTD/pull/10920
- https://github.com/OpenTTD/OpenTTD/pull/10928
- https://github.com/OpenTTD/OpenTTD/pull/10926
- https://github.com/OpenTTD/OpenTTD/pull/10934
- https://github.com/OpenTTD/OpenTTD/pull/10932
- https://github.com/OpenTTD/OpenTTD/pull/10933
- All language changes

### CODE REVIEW

File: src/lang/english.txt [hunk 1]
```diff
@@ -1530,7 +1530,7 @@ STR_CONFIG_SETTING_LINKGRAPH_COLOURS_GREY_TO_RED                :Grey to red
 STR_CONFIG_SETTING_LINKGRAPH_COLOURS_GREYSCALE                  :Greyscale
 
 STR_CONFIG_SETTING_SCROLLMODE                                   :Viewport scroll behaviour: {STRING2}
-STR_CONFIG_SETTING_SCROLLMODE_HELPTEXT                          :Behaviour when scrolling the map
+STR_CONFIG_SETTING_SCROLLMODE_HELPTEXT                          :Behaviour when scrolling the map. The "mouse position locked" options don't work on all systems, like web-based versions, touchscreens, Linux with Wayland, and others
```
Comment (rubidium42) [inline src/lang/english.txt:1533]: Do we maybe want to wait until the next run of the web translator so translators at least have a chance to update the string before the release?

Comment (TrueBrain) [reply src/lang/english.txt:1533]: We need to wait for at least 1 more eints round before we make 13.2, yes. But in general we should do a language backport just before release.
We can force an eints run early, but the question is more: when do we want to release 13.2 :)

**Comment by glx22** [comment]
As I wrote in my last minute comment in previous backport run, probably also needs #10879 since #10878 is included.
Oh #10747 (and follow-ups) would be nice too I guess.

**Comment by TrueBrain** [comment]
I assume if I now open those PRs you added the label? :) Please don't make me do that ;)

**Comment by TrueBrain** [comment]
All done; had to fix up some commits, but nothing shocking (mostly adding `c_str()` in places). Tested, it started, could play the game, so good to go!

**Review by rubidium42 — DISMISSED** [review]

**Review by rubidium42 — APPROVED** [review]

**Pull request merged**